ftgo-order-service/build.gradle

```diff
@@ -78,7 +78,7 @@ dockerCompose {
 }
 
 integrationTest.dependsOn(integrationTestsComposeUp)
-
+componentTestsComposeUp.dependsOn(integrationTestsComposeUp)
```

Comment:
Why is this necessary? The ftgo-order-service container `depends_on` mysql

Reply:
It adds additional timeout to connect to mysql. Otherwise mysql is unavailable.

Reply:
Two things.

1. Why is that a problem? It becomes available eventually
2. I think explicitly including `mysql` as a startedService of componentTest configuration will cause Docker Compose plugin to wait.

Reply:
1) Order service fails to start because mysql is not available initially.
2) I will try

Reply:
fix from point 2 did not help